pkg/messages/messages_en.go

```diff
@@ -143,6 +143,12 @@ var messagesEn = map[string]Message{
 		Type:        codes.Internal,
 		Action:      "Please check if there is any error in POD describe related with volume attach",
 	},
+	DevicePathNotFound: {
+		Code:        DevicePathNotFound,
+		Description: "Device path '%s' is not present",
+		Type:        codes.Internal,
+		Action:      "Please check if there is any error in POD describe related with volume attach",
```

Comment:
In this case we need to transfer this defect to VPC, so we can ask user to confirm by using `ibmcloud ks storage attachment` command and say if volume is attached then user can open the ticket for vpc